src/Maestro/MaestroApplication/ApplicationPackageRoot/ApplicationManifest.xml

```diff
@@ -57,6 +57,7 @@
     <ResourceOverrides>
       <Endpoints>
         <Endpoint Name="ServiceEndpoint" Port="[WebPort]" Protocol="[WebProtocol]" />
+        <Endpoint Name="ServiceEndpointHttp" Port="8088" Protocol="http"/>
```

Comment:
How does this work if we don't override the parameter above? We'd get 2x http? Will it break?

Maybe we just need to stop overriding the default http parameter in the `Int.xml` etc?

Reply:
If we change the "default" parameters in the ApplicationManifest, then we'll have an https endpoint when doing Local deployment,

Reply:
> We will have two http endpoints listening on different ports, so it should be fine

I think it will crash. Cannot listen twice on the same port.

> If we change the "default" parameters in the ApplicationManifest, then we'll have an https endpoint when doing Local deployment,

I mean we change `src\Maestro\MaestroApplication\ApplicationParameters\Int.xml`